code/cutscene/cutscenes.h

```diff
@@ -13,12 +13,16 @@
 #include "globalincs/globals.h"
 #include "globalincs/pstypes.h"
 
+#define CF_VIEWABLE			(1<<0)
+#define CF_ALWAYS_VIEWABLE	(1<<1)
+#define CF_NEVER_VIEWABLE	(1<<2)
```

Comment:
Even though the number of flags is limited, I think it would be better to use a `flagset` just for type safety, although I don't feel strongly enough about it to request changes.

Reply:
Noted. I would like to dig into flagset functionality in the future, but I stuck with good old C bitfields for this PR because I'm pretty swamped these days.

Reply:
I understand not wanting to convert this to a flagset but please use an `enum` to get rid of the preprocessor defines.

Reply:
Why `enum`?  Do you mean `const`?  An enum is the least suitable option because enums are for distinct values and flags can overlap.

Reply:
Enums are routinely used for bitflags since they are very similar to macro constants without all the namespacing issues.

Reply:
This was getting super annoying, so I just changed everything to a flagset.